ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -159,6 +159,205 @@ objects:
         mkdir -p /tmp/output
         cd /tmp/output
 
+        function restore-cluster-state() {
+          echo "Placing file /etc/rollback-test with contents A"
+          cat > /tmp/machineconfig.yaml <<'EOF'
+          apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
I think you need to dedent this and the remaining here-document lines to the script level.  For example:

```console
$ function testing() {
>   cat <<EOF
>   hello
>   world
> EOF
> }
$ testing
```

unless we switch to [`<<-EOF`][1] and tab indents.  With the space-indents for the YAML itself, it's probably best to stick with `<<EOF` and just dedent the content.

[1]: https://pubs.opengroup.org/onlinepubs/9699919799.2018edition/utilities/V3_chap02.html#tag_18_07_04

Reply:
@vrutkovs: still need this content dedent, right?

Reply:
Not sure, somehow its fine for `oc`